pkg/controller/controller_pod.go

```diff
@@ -129,12 +129,13 @@ func (tc *TFJobController) reconcilePods(
 		tfjob.Status.TFReplicaStatuses[rtype] = &tfv1alpha2.TFReplicaStatus{}
 	}
 
-	tfjob.Status.TFReplicaStatuses[rtype].Active = int32(len(activePods))
+	tfjob.Status.TFReplicaStatuses[rtype].Active -= int32(diff)
```

Comment:
Can you add some comments about this logic?

Reply:
OK, will do it.